src/router.js

```diff
@@ -6,10 +6,10 @@ const apiRouter = require("./routers/apiRouter");
 const router = express.Router();
 
 function requireTeacher(req, res, next) {
-  if (req.session.isTeacher) {
+  if (true) {
```

Comment:
Hmmmm

Reply:
Yes dit had ik gedaan om op elke pagina te komen zonder in te loggen